### PR TITLE
Added github actions to replace travis

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2103

### CODE REVIEW

**Review by UsamaSadiq — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: CI
```
Comment (UsamaSadiq) [inline .github/workflows/ci.yml:1]: It should be `Python CI` to match the URL in `ci-badge` status.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        env:
+          TOXENV: ${{ matrix.toxenv }}
+        run: tox
+
+      - name: Run Coverage
+        if: matrix.python-version == '3.8' && matrix.toxenv=='django22'
+        uses: codecov/codecov-action@v1
+        with:
+          flags: unittests
+          fail_ci_if_error: true
```
Comment (mraarif) [inline .github/workflows/ci.yml:46]: the file should end with a blank line

File: Makefile [hunk 3]
```diff
@@ -66,7 +67,10 @@ quality: ## check coding style with pycodestyle and pylint
 pii_check: ## check for PII annotations on all Django models
 	tox -e pii_check
 
-requirements: clean_pycrypto piptools dev_requirements ## install pip tools and dev requirements
+requirements: clean_pycrypto pip piptools dev_requirements ## install pip tools and dev requirements
+
+pip:
+	pip install -qr requirements/pip.txt
```
Comment (mraarif) [inline Makefile:73]: should reformat this to:
```
pip:
    pip install -qr requirements/pip.txt
```

**Review by mraarif — COMMENTED** [review]
we should create a follow-up PR to remove `travis.yml` and a few other suggestions

**Review by mraarif — APPROVED** [review]

**Pull request merged**